packages/layout/src/index.js

```diff
@@ -0,0 +1,3 @@
+export * from './react/index.js'
+export * from './vars/index.js'
+export * from './css/index.js'
```

Comment:
I think I'd namespace the css like the other packages, ie:

`export { * as css } from './css/index.js'`

Does that look right?  lol

And then once I did that, I think I'd renamed things like `asideLayoutCSS` to `asideLayout`.  

I'd do this because it more closely mimics other packages.  What do you think?

Reply:
I'm tempted to do that with the vars too.  The problem there is that `AsideLayout` (react component) and `asideLayout` (var object) are so close in naming (only differing in case).  

But if we did that, the problem created would be that it's *less* consistent with the other packages.

This makes me want to simply combine `vars/*.js` into a single `vars/index.js`.  With some var renaming (eg, `const counts` -> `const equalColumnLayoutCounts`), that would work just fine.

Reply:
Similar to position package from an earlier PR devs seem to be familiar with camel for functions/vars and PascalCase for react components. However if this proves to be an issue I say we reevaluate. Also  think maybe adding typings would help if more teams start moving to typescript.